assets/scripts/script.js

```diff
@@ -6,11 +6,16 @@
 const implex = (function () {
   const START_TRANSITION_CLASS = 'start';
   const HEADER_ID = 'header';
+  const defaultText = 'File size up to 10MB';
```

Comment:
Please, rename to smth more meaningful. There can be a lot of default texts in app